### PR TITLE
Drop client id

### PR DESCRIPTION
**Pull request by taion**
cc @edvinerikson

Relates to https://github.com/edvinerikson/relay-subscriptions/issues/6#issuecomment-241234891.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -26,8 +26,11 @@ describe('default resolution', () => {
       subscription: new GraphQLObjectType({
         name: 'Subscription',
         fields: {
-          foo: subscriptionWithClientId({
+          foo: subscription({
             name: 'FooSubscription',
+            inputFields: {
+              dummy: { type: GraphQLString },
```
Comment (taion) [inline test/index.js]: Empty objects aren't allowed in GraphQL object types, so dropping `clientSubscriptionId` means that users have to specify a dummy input field for these cases.

Comment (edvinerikson) [inline test/index.js]: If Relay doesn't check that when it's printing subscriptions/operations maybe we can change that.

Comment (edvinerikson) [inline test/index.js]: https://github.com/facebook/relay/blob/master/src/traversal/printRelayOSSQuery.js#L132

Comment (taion) [inline test/index.js]: It's this check in graphql-js: https://github.com/graphql/graphql-js/blob/a819fb3a575f27d077a460f9a4327610b87abe40/src/type/definition.js#L911-L916

And then https://github.com/facebook/relay/blob/master/scripts/babel-relay-plugin/lib/RelayQLPrinter.js#L272 -> https://github.com/facebook/relay/blob/master/scripts/babel-relay-plugin/lib/RelayQLPrinter.js#L643-L645 in babel-relay-plugin that enforce that subscriptions take a single `input` argument. Though I guess the default could be a nullable string or something.

**Comment by taion** [comment]
I think https://github.com/taion/graphql-relay-subscription/pull/4 might make more sense.

The Relay plugin will actually always add `clientSubscriptionId`

**Pull request closed without merging**